examples/meta/src/classifier/multiclass_machine.sg.in

```diff
@@ -0,0 +1,37 @@
+CSVFile f_feats_train("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
```

Comment:
we need to avoid making the folder structure more messy. There is a "multiclass" folder and a "classifier" folder. Have a look and re-decide where this example best fits

Reply:
Hmm I think this example could fit more in the multiclass folder but future classifier examples could be more suited in `classifier`